packages/grid/x-data-grid/src/tests/columns.spec.tsx

```diff
@@ -42,8 +42,8 @@ function RenderCellParamsExplicitTyping() {
         {
           field: 'price6',
           type: 'actions',
-          // @ts-expect-error `price` is expected to be a number because of GridColDef
-          getActions: (params: GridRowParams<{ price: string }>) => {
+          getActions: (params: GridRowParams<{ price: number }>) => {
+            // @ts-expect-error `toUpperCase` doesn't exist in number
```

Comment:
TS [was failing](https://app.circleci.com/pipelines/github/mui/mui-x/40278/workflows/9ef36fe7-9afa-401c-985a-acdd076fd743/jobs/231111) on this line.
@mui/xgrid What do you think about this change?
Does it make sense? 🤔

Reply:
This assertion doesn't make much sense currently. I think the intention was to be able to pass a row model to `GridRowParams`. I'll push an update to this branch, thanks for the heads-up!